src/clr-angular/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -3,141 +3,109 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Component, ContentChild, ContentChildren, OnDestroy, OnInit, QueryList } from '@angular/core';
-import { Subscription } from 'rxjs';
+import { Component, ContentChild, ContentChildren, Inject, QueryList } from '@angular/core';
 
 import { Point } from '../../popover/common/popover';
 
 import { ClrDatagridColumnToggleButton } from './datagrid-column-toggle-button';
 import { ClrDatagridColumnToggleTitle } from './datagrid-column-toggle-title';
-import { DatagridHideableColumnModel } from './datagrid-hideable-column.model';
-import { ColumnToggleButtonsService } from './providers/column-toggle-buttons.service';
-import { HideableColumnService } from './providers/hideable-column.service';
+
 import { ClrCommonStrings } from '../../utils/i18n/common-strings.interface';
+import { ColumnsService } from './providers/columns.service';
+import { DatagridColumnState } from './interfaces/column-state.interface';
+import { BehaviorSubject } from 'rxjs/index';
```

Comment:
You've got the wrong import paths on a few rxjs imports like this one.

Reply:
done